addons/OPT/GPS/fn_clientInitEH.sqf

```diff
@@ -0,0 +1,51 @@
+#include "macros.hpp";
+
+["Killed", {
+    DUMP("KILLED");
+    DUMP(_this);
+	params ["_unit", "_killer", "_instigator", "_useEffects"];
+    _unit = _unit select 0;
```

Comment:
Finde ich verwirrend hier den selben Variablennamen zu verwenden. Insgesamt wäre glaub eh ein Kommentar darüber angebracht, was in dem `_unit`-Array drin steht. Vom Namen würde man nämlich eine einzelne EInheit erwarten. Da dem nicht so ist wäre es für zukünftige Code-Leser von Vorteil diese Info hier gleich zu dokumentieren :)